### PR TITLE
Opsfiles enhancements

### PR DESCRIPTION
**Pull request by rosenhouse**
Simplify, fix up ops-files.

### CODE REVIEW

File: deploy/cf-deployment-operations/add-istio.yml [hunk 1]
```diff
@@ -108,11 +110,9 @@
       query: "*"
 
 - type: replace
-  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/app_domains?
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/app_domains/-
   value:
-    - name: "apps.internal"
```
Comment (zachgersh) [inline deploy/cf-deployment-operations/add-istio.yml:113]: If we remove this we won't have an internal apps domain anymore and I am pretty sure CI will go red. Was this intentional?

File: deploy/cf-deployment-operations/add-istio.yml [hunk 2]
```diff
@@ -54,6 +54,8 @@
     jobs:
     - name: envoy
       release: istio
+      consumes:
+        pilot: {from: pilot}
```
Comment (zachgersh) [inline deploy/cf-deployment-operations/add-istio.yml:58]: Just as an FYI, what are we actually consuming from pilot here?

File: deploy/cf-deployment-operations/add-route-syncer.yml [hunk 3]
```diff
@@ -1,29 +1 @@
...
-         databases:
-         - name: cloud_controller
-           tag: cc
-         db_scheme: mysql
-         port: 3306
-         roles:
-         - name: cloud_controller
-           password: ((cc_database_password))
-           tag: admin
+# deprecated.  this functionality is now part of add-istio.yml
```
Comment (zachgersh) [inline deploy/cf-deployment-operations/add-route-syncer.yml:1]: deprecating this was AWESOME. Glad to see this style continue

**Review by zachgersh — CHANGES_REQUESTED** [review]
Just one thing to look at here around internal domain - I can merge after that.

File: deploy/cf-deployment-operations/add-istio.yml (hunk 1 above)
Comment (rosenhouse) [reply deploy/cf-deployment-operations/add-istio.yml:113]: See line 113: I'm changing a list replace to a list append.  the `apps.internal` is already there, added by the ops-file for [enable-service-discovery](https://github.com/cloudfoundry/cf-deployment/blob/master/operations/enable-service-discovery.yml)

Comment (zachgersh) [reply deploy/cf-deployment-operations/add-istio.yml:113]: perfect, thanks for clarifying.

**Review by zachgersh — APPROVED** [review]

**Comment by rosenhouse** [comment]
Thanks!!

**Pull request merged**